src/components/common/ColonyActions/helpers/mapItemToMessageFormat.tsx

```diff
@@ -235,5 +237,50 @@ export const mapColonyActionToExpectedFormat = (
         ActionTitleMessageKeys.SafeTransactionTitle,
         notMaybe(actionData.metadata?.customTitle),
       ),
+    [ActionTitleMessageKeys.RecipientsNumber]: expenditureData?.slots?.reduce(
+      (
+        result: {
+          recipients: string[];
+        },
+        slot,
+      ) => {
+        const { recipientAddress } = slot;
+
+        if (
+          typeof recipientAddress === 'string' &&
+          !result.recipients.includes(recipientAddress)
+        ) {
+          result.recipients.push(recipientAddress);
+        }
+
+        return result;
+      },
+      {
+        recipients: [],
+      },
+    ).recipients.length,
```

Comment:
Could be simplified to:

```suggestion
     [ActionTitleMessageKeys.RecipientsNumber]: new Set(
      expenditureData?.slots.map((slot) => slot.recipientAddress),
    ).size,
```